features/manage-position/Create.tsx

```diff
@@ -73,6 +78,22 @@ const Create = () => {
   } = empState;
   const liquidationPriceWarningThreshold = 0.1;
 
+  useEffect(() => {
+    if (gcr !== null) {
+      setBackingCollateralToMin(gcr, Number(tokens));
+    }
+  }, [gcr, tokens]);
+
+  const setBackingCollateralToMin = (_gcr: number, _tokens: number) => {
+    // By default set amount of collateral to the minimum required by the GCR constraint. This
+    // default is intended to encourage users to maximize their capital efficiency.
+    const minBackingCollateral = _gcr * _tokens;
+    // We want to round down the number for better UI display, but we don't actually want the collateral
+    // amount to round down since we want the minimum amount of collateral to pass the GCR constraint. So,
+    // we'll add a tiny amount of collateral to avoid accidentally rounding too low.
+    setCollateral((minBackingCollateral + 0.00005).toFixed(4));
```

Comment:
what are the implications of hard coding this if we have non-18 decimal collateral type? `0.00005` ether is not much but `0.00005` is like 50c. it's also probs fine but just though i'd mention it.